### DISCUSSION TITLE
Crazyhouse

### DISCUSSION THREAD

**Discussion by CPagador** [fairy-stockfish/Fairy-Stockfish#536] [General]
Hi,

I'd like to use the strongest engine for crazyhouse in Windows, so I tried the BMI2 8x8 binary clicking on the following link:

https://github.com/ianfab/Fairy-Stockfish/releases/latest/download/fairy-stockfish_x86-64-bmi2.exe

It does not work, please fix it.

I got there from https://fairy-stockfish.github.io/download/

I could download the largeboard one, though.

Best regards,
Carlos

**Comment by ianfab** [comment]
The links from the website just refer to the github releases, so you can always directly download from https://github.com/ianfab/Fairy-Stockfish/releases as well. The 8x8 links just don't work right now since the latest release was a special one and the first and last one of that kind, so this won't happen any more after the next release. The last normal release 14.0 has 8x8 binaries. Alternatively development versions can be downloaded from https://github.com/ianfab/Fairy-Stockfish/actions/workflows/release.yml?query=branch%3Amaster+is%3Asuccess as well if desired.

Edit: Coincidentally just today github published https://github.blog/changelog/2022-10-21-explicitly-set-the-latest-release/, so I relabeled 14.0 as the "latest" release, so the links should now hopefully redirect correctly.
